rllib/algorithms/algorithm_config.py

```diff
@@ -152,13 +160,18 @@ def __init__(self, algo_class=None):
         }
 
         # `self.multi_agent()`
-        self.policies = {}
+        self._is_multi_agent = False
+        self.policies = {DEFAULT_POLICY_ID: PolicySpec()}
         self.policy_map_capacity = 100
         self.policy_map_cache = None
-        self.policy_mapping_fn = None
-        self.policies_to_train = None
+        self.policy_mapping_fn = (
+            lambda aid, episode, worker, **kwargs: DEFAULT_POLICY_ID
+        )
+        self.policies_to_train = [DEFAULT_POLICY_ID]
         self.observation_fn = None
         self.count_steps_by = "env_steps"
+        self._multi_agent_legacy_dict = {}
+        self._set_ma_legacy_dict()
```

Comment:
Shim for backward compatibility, iff users still access the "multiagent" key inside a AlgorithmConfig object and poke around in that dict (e.g. access config["multiagent"]["policies"]).